test-harness/build.gradle

```diff
@@ -1,5 +1,7 @@
 apply plugin: 'groovy'
 
+ext['elasticsearch.version'] = revElasticSearch6
```

Comment:
This is not needed here.

Reply:
please, see [previous comment](https://github.com/Netflix/conductor/pull/2324#discussion_r680390401) about a preferable way to avoid transitive dependencies conflicts.

Reply:
removed